.mob-ai-filename

```diff
@@ -0,0 +1 @@
+mob-ai-csharp.dll
```

Comment:
This should be the filename of the source code submitted by the user, not the binary file. But don't worry about it, I'll fix this part when I get to test it.

Reply:
Thanks!